### PR TITLE
Responsive iframe

### PR DESCRIPTION
**Pull request by danielstreif**
**What**:
- make iframe responsive for better ux
- not ideal for some very small screen sizes in portrait mode but works well besides that

**Checklist**:

- [x] Ready to be merged

### CODE REVIEW

File: src/views/calendar.hbs [hunk 1]
```diff
@@ -2,5 +2,6 @@
 <title>{{{__ "calendar_title"}}} – {{{__ "common_title"}}}</title>
 {{/section}}
 
-<iframe width="100%" height="100%" frameborder="0" src="https://teamup.com/ksb15466a44b299f51"
-  style="overflow: hidden"></iframe>
+<div class="embed-responsive embed-responsive-4by3" style="height: 100%;">
```
Comment (pheeria) [inline src/views/calendar.hbs]: Is it still necessary to add `style="height: 100%;"`?

File: src/views/calendar.hbs [hunk 2]
```diff
@@ -2,5 +2,6 @@
...
+  <iframe class="embed-responsive" src="https://teamup.com/ksb15466a44b299f51"></iframe>
```
Comment (pheeria) [inline src/views/calendar.hbs]: According to [the docs](https://getbootstrap.com/docs/4.0/utilities/embed/), iframe should have `embed-responsive-item` class.

File: src/views/calendar.hbs (hunk 1 above)
Comment (danielstreif) [reply src/views/calendar.hbs]: Yes, makes the difference between
image and image

File: src/views/calendar.hbs (hunk 2 above)
Comment (danielstreif) [reply src/views/calendar.hbs]: Changed it

File: src/views/calendar.hbs (hunk 1 above)
Comment (jarjan) [reply src/views/calendar.hbs]: There is an existing bootstrap util class already: `.h-100`

Comment (danielstreif) [reply src/views/calendar.hbs]: Thanks, I changed it

**Pull request merged**